src/test/javascript/spec/component/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback-suggestion.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { EventEmitter } from '@angular/core';
+import { Feedback } from 'app/entities/feedback.model';
+import { Course } from 'app/entities/course.model';
+import { CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent } from 'app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback-suggestion.component';
+import { MockComponent } from 'ng-mocks';
+import { FeedbackSuggestionBadgeComponent } from 'app/exercises/shared/feedback/feedback-suggestion-badge/feedback-suggestion-badge.component';
+
+describe('CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent', () => {
+    let component: CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent;
+    let fixture: ComponentFixture<CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [FontAwesomeModule],
+            declarations: [CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent, MockComponent(FeedbackSuggestionBadgeComponent)],
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(CodeEditorTutorAssessmentInlineFeedbackSuggestionComponent);
+        component = fixture.componentInstance;
+        component.feedback = new Feedback();
+        component.course = new Course();
+        component.onAcceptSuggestion = new EventEmitter();
+        component.onDiscardSuggestion = new EventEmitter();
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
```

Comment:
```suggestion
```
I don‘t think a test like this is necessary. All the other tests would fail anyway if the component could not be initialised.